tests/components/wallbox/test_config_flow.py

```diff
@@ -59,17 +54,16 @@ async def test_form_cannot_authenticate(hass: HomeAssistant) -> None:
         DOMAIN, context={"source": config_entries.SOURCE_USER}
     )
 
-    with requests_mock.Mocker() as mock_request:
-        mock_request.get(
-            "https://user-api.wall-box.com/users/signin",
-            json=authorisation_response,
-            status_code=HTTPStatus.FORBIDDEN,
-        )
-        mock_request.get(
-            "https://api.wall-box.com/chargers/status/12345",
-            json=test_response,
-            status_code=HTTPStatus.FORBIDDEN,
-        )
+    with (
+        patch(
+            "wallbox.Wallbox.authenticate",
+            new=Mock(side_effect=http_403_error),
+        ),
+        patch(
+            "wallbox.Wallbox.pauseChargingSession",
+            new=Mock(side_effect=http_403_error),
+        ),
+    ):
```

Comment:
Let's patch them where we use them, so `homeassistant.components.wallbox....`.

Also, should we just patch out the full `Wallbox` object in a common fixture?

Reply:
1. done!
2. not getting this to work like I would like. Do you know of an integration I can use as an example? Currently I'm stuck trying to override the Mocked class methods when I want them to raise an error, so combining a Mocked class with pytest.fixture with a local patch in a function to raise a side effect.